engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -3556,7 +3556,9 @@ public void reallyRun() {
 
                 final List<Long> networkIds = _networksDao.findNetworksToGarbageCollect();
                 final int netGcWait = NumbersUtil.parseInt(_configDao.getValue(NetworkGcWait.key()), 60);
-                logger.info("NetworkGarbageCollector uses '{}' seconds for GC interval.", netGcWait);
+                final int netGcInterval = NumbersUtil.parseInt(_configDao.getValue(NetworkGcInterval.key()), 60);
+                logger.info("NetworkGarbageCollector uses '{}' seconds for GC wait.", netGcWait);
+                logger.info("NetworkGarbageCollector uses '{}' seconds for GC interval.", netGcInterval);
```

Comment:
Perhaps these 2 lines can be unified into a single one: `NetworkGarbageCollector uses '{}' seconds for GC wait and '{}' seconds for GC interval`.